tensorflow-whl/README.md

```diff
@@ -0,0 +1,22 @@
+# Build new Tensorflow wheels
```

Comment:
Was there any reason you preferred to put the Tensorflow wheel build (everything in this dir) into a separate file, where the CPU and GPU images are under the root?

Related, could you reduce both build (and test, push, ...) files (this set and the one in the parent dir) to a single one (e.g. in the parent dir)?

Another approach (if this consistency is appropriate) is to place each under a subdir, not just the Tensorflow wheel build.

Reply:
My original reasoning behind it is:
 - the docker-python repo is for building the cpu and gpu images, this is why their Dockerfiles are at the root.
- both of these images requires the tensorflow image so I put it in a subdirectory.

Given that the GPU depends on the CPU, the logic above is somewhat flawed ;)

I can see some value in flattening to reuse the scripts (build, test, push). Do you prefer that I do it in this PR or a separate PR?

Reply:
Best to do in another PR, and only if you really do prefer to change it.  Just bringing up things I notice, but you have a better handle on how this should look and I won't feel hurt :)